with-baseweb/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "private": true,
+  "sideEffects": false,
+  "scripts": {
+    "build": "remix build",
+    "dev": "remix dev",
+    "start": "remix-serve build"
+  },
+  "dependencies": {
+    "@remix-run/node": "*",
+    "@remix-run/react": "*",
+    "@remix-run/serve": "*",
+    "baseui": "^12.2.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "styletron-engine-atomic": "^1.5.0",
+    "styletron-react": "^6.1.0"
+  },
+  "devDependencies": {
+    "@remix-run/dev": "*",
+    "@remix-run/eslint-config": "*",
+    "@types/react": "^17.0.39",
+    "@types/react-dom": "^17.0.13",
```

Comment:
```suggestion
    "@types/react": "^18.0.25",
    "@types/react-dom": "^18.0.8",
```

Reply:
We'll keep React 17 for now (see https://github.com/remix-run/examples/pull/56#issuecomment-1314298186)